docs/sdk-json-proposal/SDK_API_EXAMPLES.md

```diff
@@ -0,0 +1,199 @@
+# Mesh Data Plane SDK – Examples
+
+## Video Sender Example
+
+```c
+int main(void)
+{
+    /* Default client configuration */
+    const char *client_config = R"(
+      {
+        "apiVersion": "v1",
+        "apiConnectionString": "Server=192.168.96.1; Port=8001",
```

Comment:
Can you split "apiConnectionString" to key-value pairs ?

Reply:
Consider similar cases in the industry https://www.connectionstrings.com/sql-server/